packages/office-ui-fabric-react/src/components/Persona/Persona.test.tsx

```diff
@@ -0,0 +1,56 @@
+/* tslint:disable-next-line:no-unused-variable */
+import * as React from 'react';
+import { setRTL } from '../../utilities/rtl';
+import { Persona, PersonaInitialsColor } from './index';
+import { shallow } from 'enzyme';
+import * as chai from 'chai';
+
+const { expect } = chai;
+
+describe('Persona', () => {
+  describe('initials and colors', () => {
+    beforeEach(() => {
+      setRTL(false);
+    });
```

Comment:
consider splitting out the initials logic into a utility that can be separated out, so that it can be unit tested.

But at a minimum if you could add a few more test cases, that would be good because the logic/regexes are pretty complex.

E.g. what initials are generated by:

Empty primaryText
Onename
one-name
David Foo-Bar
David Blah Blah Blah
Mr. David Zearing
Dr. Zearing
(David)
4lex 4loo